core/src/test/java/com/pholser/junit/quickcheck/runner/PropertyFalsifiedUtilityClassTest.java

```diff
@@ -26,11 +26,119 @@ a copy of this software and associated documentation files (the
 package com.pholser.junit.quickcheck.runner;
 
 import com.pholser.junit.quickcheck.UtilityClassesUninstantiabilityHarness;
+import org.junit.*;
+
 
 public class PropertyFalsifiedUtilityClassTest
     extends UtilityClassesUninstantiabilityHarness {
 
     public PropertyFalsifiedUtilityClassTest() {
         super(PropertyFalsified.class);
     }
+
```

Comment:
A couple of things on these tests:

1) Please static-import `assertEquals`.

2) Can you separate the tests into "arrange", "act", and "assert" sections with single blank lines, as described [here](http://wiki.c2.com/?ArrangeActAssert)?